File: Codeforces Round #691(Div. 2)/Python Solutions/A_Red-Blue_Shuffle.py

t = int(input())
for _ in range(t):
    n = int(input())
    r = input()
    b = input()
    r_count = 0
    b_count = 0
    for i in range(n):
        if r[i] > b[i]:
            r_count += 1
        elif b[i] > r[i]:
            b_count += 1
    if r_count > b_count:
        print('RED')
    elif b_count > r_count:
        print('BLUE')
    else:
        print('EQUAL')

File: Codeforces Round #691(Div. 2)/C++ Solutions/A_Red-Blue_Shuffle.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main() {
ios_base::sync_with_stdio(false);
cin.tie(NULL);

int t;
  cin >> t;
  while(t--){
    int n;
    cin >> n;
    string r, b;
    cin >> r;
    cin >> b;
    int red=0;
    int blue=0;
    for(int i=0; i<n; i++){
      if((int)r[i] > (int)b[i]) red++;
      if((int)r[i] < (int)b[i]) blue++;
    }
    if(red>blue) cout << "RED\n";
    else if(red<blue) cout << "BLUE\n";
    else cout << "EQUAL\n";
  }
return 0;
}
